openhcl/hcl/src/ioctl.rs

```diff
@@ -1577,6 +1577,35 @@ impl<'a, T: Backing> ProcessorRunner<'a, T> {
         }
     }
 
+    /// Gets the kernel offload interrupt request bitmap.
+    pub fn kernel_ipi_offload_irr(&mut self) -> Option<[u32; 8]> {
+        // SAFETY: the `ipi_offload_irr` field of the run page
+        // is atomically accessed by everyone.
```

Comment:
i'd clarify what everyone is (in this case, different CPUs and kernel mode)

Reply:
Also, this isn't a real safety comment, why it needs to say _why_ this operation is safe. Is it only the offload_irr cast? It seems like that's the only part that actually requires unsafe?

Reply:
Equivalent comment in the new version should be better.